src/sentry/api/urls.py

```diff
@@ -1121,6 +1122,12 @@
                     OrganizationJoinRequestEndpoint.as_view(),
                     name="sentry-api-0-organization-join-request",
                 ),
+                # relay usage
+                url(
+                    r"^(?P<organization_slug>[^\/]+)/relay_history/$",
```

Comment:
_nit_: Since we call the model `RelayUsage`, this could also be called `/relay_usage/`. But I'm happy with `history`, too.

Reply:
relay_usage works for me